test/runtests.jl

```diff
@@ -1,3 +1,6 @@
+using MATLAB
+using Base.Test
```

Comment:
I think I prefer to keep them like this, since this seems to follow the Julia convention. If you want to test individual files, just comment them out in the `runtests` file that way you can still take advantage of `Pkg.test` ?

I think it was also mentioned that in the future we will be able to run tests files individually.